apis/projectcontour/v1/httpproxy.go

```diff
@@ -567,6 +619,9 @@ type Service struct {
 	// Rewriting the 'Host' header is not supported.
 	// +optional
 	ResponseHeadersPolicy *HeadersPolicy `json:"responseHeadersPolicy,omitempty"`
+	// The policies for rewriting Set-Cookie header attributes.
+	// +optional
+	CookieRewritePolicies []CookieRewritePolicy `json:"cookieRewritePolicies,omitempty"`
```

Comment:
Will Envoy end up processing these in the order specified? A small comment here to ensure that's clear would be useful.

Reply:
multiple policies on the same cookie are not allowed, so i think order should not matter?